bigframes/_config/sampling_options.py

```diff
@@ -33,14 +33,44 @@ class SamplingOptions:
     random_state: Optional[int] = None
 
     def with_max_download_size(self, max_rows: Optional[int]) -> SamplingOptions:
+        """Configures the maximum download size for data sampling in MB
+
+        Args:
+            max_rows (None or int):
+                An int value for the maximum row size.
+
+        Returns:
+            SamplingOptions:
```

Comment:
Please use the full path as a customer would use it. Otherwise, Cloud RAD might not find it. In this case `bigframes._config.sampling_options.SamplingOptions`. That way it can link to https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes._config.sampling_options.SamplingOptions

Note: in this case it's a bit less essential as it's on the same page, but I'd like us to set a good example for others to follow.